### PR TITLE
Expose params

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

**Comment by sjanssen2** [comment]
We are now stating "SEPP has two main parameters. In the default version used for Greengenes and incorporated into QIIME2, the reference tree is divided into 62 “placement” subsets, each with at most 5000 leaves, and each placement subset is further divided into alignment subsets of at most 1000 leaves to build the HMM ensamples (292 alignment subsets in total)." in the manuscript.

Thus, we need to actually expose those two expert settings to the user.

I don't see how to unit test those parameters, because they don't directly leave traces in the outputs and it is still unclear how they impact results.

**Comment by sjanssen2** [comment]
@antgonza @josenavas I would be happy for your reviews. This one should be quick and easy.

File: q2_fragment_insertion/_insertion.py [hunk 1]
```diff
@@ -133,6 +135,8 @@ def _run(seqs_fp, threads, cwd,
 
 def sepp(representative_sequences: DNASequencesDirectoryFormat,
          threads: int=1,
+         alignment_subset_size: int=1000,
```
Comment (antgonza) [inline q2_fragment_insertion/_insertion.py]: Could you add some documentation of why/how these values were set?

**Comment by sjanssen2** [comment]
@antgonza can you please check if I addressed you comment sufficiently?

**Comment by antgonza** [comment]
Thanks, the parameters are now explained but could you add why 1000/5000 are used? Is this the recommendation from the tutorials? Note that I checked but couldn't find those values ...

**Comment by sjanssen2** [comment]
All I got from Siavash is this vague description:
SEPP parameters. SEPP has two main parameters. In the default version used for Greengenes and incorporated into QIIME2, the reference tree is divided into 62 “placement” subsets, each with at most 5000 leaves, and each placement subset is further divided into alignment subsets of at most 1000 leaves to build the HMM ensamples (292 alignment subsets in total). These choices are driven by computational constraints; increasing the placement subset size (which is in theory desirable) puts a high burden on the memory, and reducing the alignment subset could increase the running time with very little improvement in the accuracy of results (Mirarab et al. 2012).

**Comment by antgonza** [comment]
Perhaps adding that would be a good idea so the next dev knows from where those values come from and/or who to contact if we need to change ...

**Comment by sjanssen2** [comment]
@antgonza I added this info as a comment in front of the method where default values are defined. Is that OK? I don't want to expose this to the end user to directly.

**Comment by sjanssen2** [comment]
@antgonza many thanks!

**Pull request merged**